documentation/adoc/cluster-controller.adoc

```diff
@@ -247,10 +247,11 @@ The JSON representation of the 'topic-controller-config` has no mandatory fields
 The configurable fields are the following :
 
 `image`:: Docker image to use for the topic controller. Default is `strimzi/topic-controller:latest`
-`namespace`:: The Kubernetes namespace (OpenShift project) in which the topic controller watches for topic ConfigMaps.
+`watchedNamespace`:: The Kubernetes namespace (OpenShift project) in which the topic controller watches for topic ConfigMaps.
 Default is the namespace where the topic controller is running
 `reconciliationIntervalMs`:: The interval between periodic reconciliations in milliseconds. Default is 900000 (15 minutes).
 `zookeeperSessionTimeoutMs`:: The Zookeeper session timeout in milliseconds. Default is 20000 milliseconds (20 seconds).
+`topicMetadataMaxAttempts`:: The number of attempts for getting topics metadata. Default `6`.
```

Comment:
TBH, I'm not sure the user would know what should he/she understand under this:
- It is for getting topics metadata from Kafka, right? -> that should be probably mentioned there. (as in _The number of attempts for getting topics metadata **from Kafka**._)
- Since we configure only the number of attempts, we should probably explain what the time between them is (or if it is exponential back-off then we should mention it).
- We might also want to explain when to change this. Either in the FAQ or when mentioning the values here and in TC docs. (i.e. _When creating large topics with many partitions / replicas_)

Reply:
In the FAQ I see a "Cluster Controller" section. Do we want to add it in a specific "Topic Controller" section or inside the "Cluster Controller" one but related on topic controller configuration ?

Reply:
TBH, I would just add it as a next sentence to both parts where you explain the option.

Reply:
Done ;)